### ISSUE TITLE
Integrate API Gateway script into Nightly CI Deploys

### ISSUE THREAD

**Issue by galenatjpl** [unity-sds/unity-cs#124]
Creation of empty REST API, into the API Gateway, so that SPS endpoints can then be added/utilized later.

See also:

https://github.com/unity-sds/unity-cs/tree/api-gateway-terraform/terraform/terraform-api-gateway

Acceptance Criteria:

- [x] Verify the REST API gets deployed to the API Gateway
- [ ] Verify specific SPS API routes get populated into the REST API as part of the nightly.
- [ ] For this phase, the SPS endpoints that need to be publicly accessible from the perspective a human and/or service API, need to be properly routed/exposed.

**Comment by ramesh-maddegoda** [comment]
Created pull request https://github.com/unity-sds/unity-cs-infra/pull/12 to add terraform scripts of API Gateway and Cognito user pool

**Comment by ryanhunter-jpl** [comment]
This is dependent on [Project account API Gateway creation](https://github.com/unity-sds/unity-cs/issues/200) which is currently blocked on [this MCP ticket](https://jaas.gsfc.nasa.gov/servicedesk/customer/portal/2/GSD-2069).

**Comment by ryanhunter-jpl** [comment]
Incorporated in nightly - post QSR I'll be merging the SPS API GW changes to test the SPS deployment side of this.

**Comment by galenatjpl** [comment]
@ramesh-maddegoda and @ryanhunter-jpl , is this ticket basically complete, but just waiting on a end to end test to verify it actually creates the API GW routes?  The blockers mentioned above on Apr 19 seemed to be resolved now.

**Comment by galenatjpl** [comment]
This ticket is essentially already covered by issue #222 and #233 .  This is redundant, so closing.
